sample-compose/app/build.gradle.kts

```diff
@@ -87,6 +88,15 @@ android {
         compose = true
     }
 
+    packagingOptions {
+        resources {
+            excludes += "/META-INF/{AL2.0,LGPL2.1,*.md}"
+        }
+        jniLibs {
+            useLegacyPackaging = true
+        }
```

Comment:
@luongvo Would you mind explaining what's the purpose of this? 🙇

Reply:
@Tuubz To resolve "libmockkjvmtiagent.so" when running tests on device with API >= 28 https://github.com/mockk/mockk/issues/297#issuecomment-901924678